apps/root/src/services/providerService.ts

```diff
@@ -303,11 +254,11 @@ export default class ProviderService {
         window.location.reload();
       }
 
-      if (provider.getNetwork) {
-        await provider.getNetwork();
-      } else if (provider.detectNetwork) {
-        await provider.detectNetwork();
-      }
+      // if (signer.dete.) {
```

Comment:
In this file we have some pieces of code commented, I suggest erasing them before merge

Reply:
Done!